doc/src/tutorials/arch/heterogeneous_tiles.rst

```diff
@@ -0,0 +1,158 @@
+.. _heterogeneous_tiles_tutorial:
+
+Heterogeneous tiles tutorial
+============================
+
+This tutorial aims at providing information to the user on how to model sub tiles to enable ``heterogeneous tiles`` in VPR.
```

Comment:
It seems like you're using \`\` text \`\` (fixed-width text) for emphasis. Generally we've tried to use that when explicitly referring to a something from code (e.g. ``BUFIO`` below).

If you want to emphasize something I'd suggest using italics (`*heterogeneous tiles*`) or bold (`**heterogeneous tiles**`), for more details see: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html.